### PR TITLE
Pagure issue 6553

### PR DESCRIPTION
**Pull request by AffableZonkey**
Two commits below;
- bff447d - updates the documentation location for status
- c4fc019 - remove ask, darkserver, fedhosted, freemedia from wsgi/statuses.json and added registry.fpo

I doubt I have access to the private aws keys, so can't really test properly. Please let me know if I can do anything else or make any changes.

### CODE REVIEW

**Comment by AffableZonkey** [comment]
Thanks!

**Comment by AffableZonkey** [comment]
crud...sorry

File: wsgi/statuses.json [hunk 1]
```diff
@@ -3,13 +3,7 @@
...
     "global_verbose_status": "There are scheduled downtimes in progress",
     "services": {
-        "ask": {
-            "message": "Everything seems to be working.",
-            "name": "Ask Fedora",
-            "status": "good",
-            "url": "http://ask.fedoraproject.org/"
-        },
-        "authentication": {
+       "authentication": {
```
Comment (pypingou) [inline wsgi/statuses.json]: there are a few lines in this diff that show a little under-indented compare to what was here before, is this expected?

Comment (AffableZonkey) [reply wsgi/statuses.json]: Thanks pypingou! I missed that. Will fix.

**Comment by AffableZonkey** [comment]
I think I have everything lined up now. If you want me to, I can cancel this PR and submit two separate ones?

**Comment by pypingou** [comment]
Looks good to me, thanks! :)

**Comment by pypingou** [comment]
wrong button... :)

**Pull request merged**